test/validate.js

```diff
@@ -50,9 +53,70 @@ describe('Validate', function () {
                 expect(error).to.have.property('attributes');
                 expect(error).to.have.deep.property('attributes[0].name');
                 expect(error).to.have.deep.property('attributes[1].name');
-                expect(error.attributes[0]).to.be.instanceOf(ModelClass.ValidationAttributeError);
-                expect(error.attributes[0].type).to.be.equal('error');
+                expect(error.attributes[0]).to.be.instanceOf(Attribute.ValidationError);
             }
         });
     });
+
+    describe('with nested entities', function () {
+        var model,
+            ModelClass = require('./models/with-nested');
+
+        beforeEach(function () {
+            model = new ModelClass({
+                a: 'a1',
+                nested: {
+                    invalid: true
+                }
+            });
+        });
+
+        it('should report invalid nested model attribute with specific error', function () {
+
+            model.get('nested').set('invalid', false);
+
+            return model.validate().always(function (p) {
+                var error;
+
+                if (p.isFullfiled) {
+                    return Vow.reject();
```

Comment:
yes, look this comment
https://github.com/bem-node/promised-models/pull/15#discussion_r43372215
I'm working on it